go/v1beta1/storage/connector.go

```diff
@@ -49,9 +49,13 @@ type connector struct {
 	dsnLock sync.RWMutex
 }
 
-func newConnector(ctx context.Context, conf *config.Config, driver driver.Driver, cc CredentialsCreator, logger *log.Logger) (*connector, error) {
+func newConnector(ctx context.Context, conf *config.Config, driver driver.Driver, cc CredentialsCreator, logger *log.Logger, overwriteHost string) (*connector, error) {
```

Comment:
I do not like the parameter `overwiteHost`. It is better (though not the best) to have a boolean parameter to switch the reader/writer instance.